### ISSUE TITLE
Route Generator Not working

### ISSUE THREAD

**Issue by manmon42** [angular-fullstack/generator-angular-fullstack#1015]
Anytime I try and use the `yo angular-fullstack:route routeName` command, all of the necessary files are generated, hoever when I try and connect to `localhost:9000/routeName` I get redirected to the home page. I have updated the generator and npm and saw no difference. Any help would be appreciated.

**Comment by drmikeh** [comment]
First make sure that the new JavaScript files have been injected into the index.html file.  You should see something like:
  <script src="app/itemDetail/routeName.controller.js"></script>
  <script src="app/itemDetail/routeName.js"></script>

If you don't see these files, then run "grunt injector".

**Comment by manmon42** [comment]
Thanks, that fixed it!

**Comment by Awk34** [comment]
Yeah that's actually a bug that I've noticed before too. The subgenerators should also inject the new files into `index.html`.

**Comment by kingcody** [comment]
I'd say that is more of an issue with `grunt-contrib-watch` than `generator-ng-component`. The injector task should fire off when new js files are created, but sometimes its doesn't...

**Comment by cgranville** [comment]
Upon looking into this issue, I noticed that the generator is generating .js files - not .ts files. Should be generating files based on the options selected in the configuration wizard.

Upon renaming the files to .ts and running grunt injector, the scripts were injected into the application.

The only other this was that after the scripts were injected, I had to restart the grunt serve task so the files were available in the .tmp folder to preview the app.
